packages/experimental/Menu/src/MenuItem/MenuItem.types.ts

```diff
@@ -10,6 +10,8 @@ export const menuItemName = 'MenuItem';
 
 export interface MenuItemTokens extends LayoutTokens, FontTokens, IBorderTokens, IColorTokens {
   checkmarkSize?: number;
+  chevronPadding?: number;
+  chevronSize?: number;
```

Comment:
Do these apply to the submenuIndicator?

Reply:
Yeah. I can rename the tokens to be that instead.

Reply:
I was actually going to ask if we should rename the submenuIndicator slot to "chevron". Since we don't have separate MenuItem and SubmenuItem controls anymore, I think the decriptive name of "submenuIndicator" is slightly better, so renaming the control tokens would be nice.